### PR TITLE
man page: Add all new heketi-cli commands and flags in man page

### PR DESCRIPTION
**Pull request by MohamedAshiqrh**
closes: #581

Signed-off-by: Mohamed Ashiq Liyazudeen <mliyazud@redhat.com>

### CODE REVIEW

**Comment by MohamedAshiqrh** [comment]
@obnoxxx This has only the changes new command and flags.

File: doc/man/heketi-cli.8 [hunk 1]
```diff
@@ -206,6 +268,14 @@ Create a GlusterFS volume
 .fi
 .RE
 .PP
+\fB               \-\-gid\fP="0"
+.PP
+.RS
+.nf
+            Optional: Initialize volume with the specified group id
+.fi
```
Comment (obnoxxx) [inline doc/man/heketi-cli.8]: Please add s/th like "Default is 0"

File: doc/man/heketi-cli.8 [hunk 2]
```diff
@@ -206,6 +268,14 @@ Create a GlusterFS volume
 .fi
 .RE
 .PP
+\fB               \-\-gid\fP="0"
```
Comment (obnoxxx) [inline doc/man/heketi-cli.8]: should it be ="0" or =0 ?

File: doc/man/heketi-cli.8 [hunk 3]
```diff
@@ -324,6 +420,14 @@ Print response as JSON
 .fi
 .RE
 .PP
+\fB\-\-log\-flush\-frequency\fP="5s"
+.PP
+.RS
+.nf
+Maximum number of seconds between log flushes
+.fi
```
Comment (obnoxxx) [inline doc/man/heketi-cli.8]: Add "Default is 5s" or so

File: doc/man/heketi-cli.8 (hunk 2 above)
Comment (MohamedAshiqrh) [inline doc/man/heketi-cli.8]: done.

File: doc/man/heketi-cli.8 (hunk 1 above)
Comment (MohamedAshiqrh) [inline doc/man/heketi-cli.8]: done.

File: doc/man/heketi-cli.8 (hunk 3 above)
Comment (MohamedAshiqrh) [inline doc/man/heketi-cli.8]: https://github.com/heketi/heketi/issues/585 look at this really interesting.

**Comment by MohamedAshiqrh** [comment]
@heketi/dev @obnoxxx can you please review the changes?

File: doc/man/heketi-cli.8 (hunk 3 above)
Comment (obnoxxx) [inline doc/man/heketi-cli.8]: ok. :-)

File: doc/man/heketi-cli.8 [hunk 4]
```diff
@@ -420,14 +421,6 @@ Print response as JSON
...
 .RE
 .PP
-\fB\-\-log\-flush\-frequency\fP="5s"
-.PP
-.RS
-.nf
-Maximum number of seconds between log flushes
-.fi
-.RE
-.PP
```
Comment (obnoxxx) [inline doc/man/heketi-cli.8]: This removal should be subject of a separate patch, imho.

**Comment by MohamedAshiqrh** [comment]
https://github.com/heketi/heketi/pull/591
